### ISSUE TITLE
incorrect peer dependency @angular/core@~4.0.1

### ISSUE THREAD

**Issue by xmlking** [swimlane/ngx-charts#375]
with Angular 4.1.0 following warnings are displayed. please decouple dependency on angular version.    
```
warning "@swimlane/ngx-charts@5.1.2" has incorrect peer dependency "@angular/common@~4.0.1".
warning "@swimlane/ngx-charts@5.1.2" has incorrect peer dependency "@angular/core@~4.0.1".
```

**Comment by gnomeontherun** [comment]
This is fixed in master, but not yet released. Also waiting for a new release, or will end up using the master until then.

See https://github.com/swimlane/ngx-charts/blob/master/package.json#L87

**Comment by marjan-georgiev** [comment]
Fixed in 5.2.1
